test/libsolidity/smtCheckerTests/types/struct/struct_mapping.sol

```diff
@@ -15,4 +15,4 @@ contract C {
 	}
 }
 // ----
-// Warning 6328: (143-169): Assertion violation happens here.
+// Warning 4661: (143-169): Assertion violation happens here.
```

Comment:
Not sure why this switched from BMC to CHC (or the ether way around) for me. There's no typecasting there.

Reply:
On CI it still gives `6328`.

Reply:
It is my local Z3 then.